spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -149,6 +149,36 @@ to `WARNING` in `logback.xml`:
 
 ====
 
+[[configuring-rdbms]]
+== RDBMS configuration
```

Comment:
This needs to be part of other server implementations as well. may be we can split this part into a separate doc and have it added in each server implementations?